scripts/validate-docs.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+pushd docs
```

Comment:
```suggestion
pushd docs || travis_terminate 1
```

In case the docs directory doesn't exit, fast-fail.